Tests/test_imagefont.py

```diff
@@ -198,6 +201,37 @@ def test_textsize_equal(self):
             # Epsilon ~.5 fails with FreeType 2.7
             assert_image_similar(im, target_img, self.metrics["textsize"])
 
+    @pytest.mark.parametrize(
+        "text,mode,font,size,length_basic_index,length_raqm",
```

Comment:
I prefer to omit the spaces to match Pytest docs: https://docs.pytest.org/en/stable/parametrize.html

Reply:
I have also found there is also the option of using a tuple. How's this?

```suggestion
        ("text", "mode", "font", "size", "length_basic_index", "length_raqm"),
```

Reply:
Pytest isn't consistent, here's an example with spaces:
 
https://docs.pytest.org/en/stable/example/parametrize.html#apply-indirect-on-particular-arguments

Grepping the pytest docs+codebase, are 28 results matching this: 

```console
$ git grep "@pytest.mark.parametrize([\"'][a-zA-Z_]\+," | wc -l
      28
```

And of those 28, 19 have a space after the comma:
```console
$ git grep "@pytest.mark.parametrize([\"'][a-zA-Z_]\+, " | wc -l
      19
```

---


My reason for this preference is that they must match the test's own parameters. In this form, it's easy to check in the editor by highlighting one:

![image](https://user-images.githubusercontent.com/1324225/95733108-52da7200-0c8a-11eb-9650-277a773b7b74.png)

Reply:
> My reason for this preference is that they must match the test's own parameters. In this form, it's easy to check in the editor by highlighting one:

Ah, that's enough to convince me :)

I didn't think of that since my editor doesn't do this (but it parses the pytest mark and adds a highlight warning on a mismatch).